pkgdb/src/resolver/manifest-raw.cc

```diff
@@ -310,6 +310,8 @@ from_json( const nlohmann::json & jfrom, Options & opts )
                 + value.dump() );
             }
         }
+      /* Not used within pkgdb */
+      else if ( key == "cuda-detection" ) { ; }
```

Comment:
Is this any cause for concern?

Reply:
I don't think so. In order to see this manifest field you would need to edit a manifest, and you can't edit v0 manifests to include it (you're always upgraded to v1 after an edit now).